docs/reference/mapping/params/eager-global-ordinals.asciidoc

```diff
@@ -0,0 +1,74 @@
+[[eager-global-ordinals]]
+=== `eager_global_ordinals`
+
+Global ordinals is a data-structure on top of doc values, that maintains an
+incremental numbering for each unique term in a lexicographic order. Each
+term has a unique number and the number of term 'A' is lower than the
+number of term 'B'. Global ordinals are only supported with
+<<keyword,`keyword`>> and <<text,`text`>> fields. In `keyword` fields, they
```

Comment:
Good question. I guess finding the top ip addresses that hit a website is a common use-case.